addons/website_slides/views/slide_question_views.xml

```diff
@@ -1,5 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
+    <record id="slide_quizze_action" model="ir.actions.act_window">
```

Comment:
Just to follow guidelines: this should be using an xml id like slide_question_action ("model"_action uses the technical model name, with . replaced by _).